app/models/investment.rb

```diff
@@ -3,6 +3,9 @@ class Investment < ApplicationRecord
 
   attribute :value, :pence
 
+  scope :for_client, -> { where(ownership_type: OwnershipType::APPLICANT.to_s) }
```

Comment:
I think we should add joint accounts to the clients evidence requirements. (Perhaps not needed until we do partner?)